testing/src/main/java/io/split/client/testing/SplitClientForTest.java

```diff
@@ -33,76 +33,76 @@ public void registerTreatment(String feature, String treatment) {
-        return _tests.containsKey(split)
-                ? _tests.get(split)
+    public String getTreatment(String key, String featureFlag, Map<String, Object> attributes) {
+        return _tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL;
     }
 
-    public String getTreatment(Key key, String split, Map<String, Object> attributes) {
-        return _tests.containsKey(split)
-                ? _tests.get(split)
+    public String getTreatment(Key key, String featureFlag, Map<String, Object> attributes) {
+        return _tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL;
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(String key, String split) {
-        return new SplitResult(_tests.containsKey(split)
-                ? _tests.get(split)
+    public SplitResult getTreatmentWithConfig(String key, String featureFlag) {
+        return new SplitResult(_tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL, null);
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(String key, String split, Map<String, Object> attributes) {
-        return new SplitResult(_tests.containsKey(split)
-                ? _tests.get(split)
+    public SplitResult getTreatmentWithConfig(String key, String featureFlag, Map<String, Object> attributes) {
+        return new SplitResult(_tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL, null);
     }
 
     @Override
-    public SplitResult getTreatmentWithConfig(Key key, String split, Map<String, Object> attributes) {
-        return new SplitResult(_tests.containsKey(split)
-                ? _tests.get(split)
+    public SplitResult getTreatmentWithConfig(Key key, String featureFlag, Map<String, Object> attributes) {
+        return new SplitResult(_tests.containsKey(featureFlag)
+                ? _tests.get(featureFlag)
                 : Treatments.CONTROL, null);
     }
 
     @Override
-    public Map<String, String> getTreatments(String key, List<String> splits) {
+    public Map<String, String> getTreatments(String key, List<String> featureFlag) {
         Map<String, String> treatments = new HashMap<>();
-        for (String split : splits) {
+        for (String split : featureFlag) {
             treatments.put(split, _tests.containsKey(split) ? _tests.get(split) : Treatments.CONTROL);
         }
         return treatments;
     }
 
     @Override
-    public Map<String, String> getTreatments(String key, List<String> splits, Map<String, Object> attributes){
+    public Map<String, String> getTreatments(String key, List<String> featureFlag, Map<String, Object> attributes){
         Map<String, String> treatments = new HashMap<>();
-        for (String split : splits) {
```

Comment:
Should use featureFlags instead of featureFlag

Reply:
Thank you! I updated.